frontend/src/metabase-lib/v1/expressions/pratt/lexifier.ts

```diff
@@ -0,0 +1,239 @@
+import type { SyntaxNodeRef } from "@lezer/common";
```

Comment:
This file now parses the tokens from `@lezer/lr` directly, so we skip intermediate representation we had (ie. `Token).